spec/controllers/hubstats/deploys_controller_spec.rb

```diff
@@ -85,11 +111,31 @@ module Hubstats
                                   :deployed_at => "2011-02-03 03:00:00 -0500",
                                   :deployed_by => "emmasax1")
         deploys_ordered = [deploy2, deploy3, deploy1]
-        expect(Hubstats::Deploy).to receive(:order_with_timespan).and_return(deploys_ordered)
+        expect(Hubstats::Deploy).to receive_message_chain("order_with_timespan.group_by.paginate").and_return(deploys_ordered)
         get :index
         expect(response).to have_http_status(200)
         expect(response).to render_template(:index)
       end
     end
+
+    describe "#show" do
+
+      before :each do
+        create(:repo, :full_name => "sportngin/ngin")
+      end
+
+      it "should show the pull requests of the specific deploy" do
+        deploy = create(:deploy, :git_revision => "c1a2b37",
+                        :repo_id => 101010,
+                        :deployed_at => "2009-02-03 03:00:00 -0500",
+                        :deployed_by => "emmasax1")
+
+        # These fail because we're not calling the actual table in the controller; the factory girl for pull request doesn't have a deploy_id.
+        # However, it would work if a pull request did belong to a deploy
+        #get :show, id: deploy
```

Comment:
Did you migrate your test database? I think we did this with `rake db:test:prepare`?